src/js/track.js

```diff
@@ -44,7 +44,29 @@ Track.prototype.getSelectedSong = function() {
 };
 
 Track.prototype.normalizeSearchQuery = function(query) {
-	var normalized = query;
+	var removeableTags = ["official music video", "lyric video",  "out now",  "cover art",  "official video", "hd", "hq", "lyrics", "radio edit", "radio mix"];
+
+	var normalized = query.toLowerCase(); // let's convert the title to lowercase for a better beginning
+
+	angular.forEach(removeableTags, function(value, key) {
+		normalized = normalized.replace(value, ''); // remove founded tag
```

Comment:
We may want to consider checking against these strings if they are wrapped in parens, or brackets. Something like `(\(|\[|\{)official video(\)|\]|\})` instead of just the string `official video`